src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -363,11 +363,23 @@ class DialogmoteService(
         referat: NewReferatDTO,
         token: String,
     ): Boolean {
+        val narmesteLeder = narmesteLederClient.activeLeader(
+            personIdentNumber = dialogmote.arbeidstaker.personIdent,
+            virksomhetsnummer = dialogmote.arbeidsgiver.virksomhetsnummer,
+            token = token,
+            callId = callId
+        )
+        if (narmesteLeder == null) {
+            throw RuntimeException("Denied access to Dialogmoter: No NarmesteLeder was found for person")
```

Comment:
Legge til en log her?

Reply:
Fikset nå (men antar at dette vil være en sjelden feilsituasjon for referat siden det har blitt laget en innkalling ikke så lenge før)